x-pack/packages/ai-infra/inference-common/src/chat_complete/api.ts

```diff
@@ -6,16 +6,35 @@
  */
 
 import type { Observable } from 'rxjs';
-import type { ToolOptions } from './tools';
+import type { ToolCallsOf, ToolOptions } from './tools';
 import type { Message } from './messages';
-import type { ChatCompletionEvent } from './events';
+import type { ChatCompletionEvent, ChatCompletionTokenCount } from './events';
 
 /**
  * Request a completion from the LLM based on a prompt or conversation.
  *
- * @example using the API to get an event observable.
+ * By default, The complete LLM response will be returned as a promise.
+ *
+ * @example using the API in stream mode to get an event observable.
```

Comment:
I think this title doesn't follow the code sample.

```suggestion
 * @example using the API in the default mode to get a Promise.
```